src/App.js

```diff
@@ -24,7 +23,6 @@ function App() {
   }
 
   return (
-    <dataContext.Provider value={{ weather, setWeather }}>
```

Comment:
We didn't need this here, as you can just pass props from App to Weather and Location. For reference, you can think that everything **below** `return` is below the level of `App`. It would look like this
```
           App
Location          Weather
```
Both can receive props from App, so it's better to just pass them like so.